samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs

```diff
@@ -133,17 +131,40 @@ public class SemanticKernelController : ControllerBase
 
         return this.Ok(new AskResult { Value = result.Result, Variables = result.Variables.Select(v => new KeyValuePair<string, string>(v.Key, v.Value)) });
     }
-    private async Task RegisterOpenApiSkillsAsync(OpenApiSkillsAuthHeaders openApiSkillsAuthHeaders, IKernel kernel)
+
+    /// <summary>
+    /// Register authenticated OpenAPI skills with the planner's kernel.
+    /// If the request includes an auth header for an OpenAPI skill, register the skill with the kernel.
+    /// Else, don't register the skill as it'll fail on auth.
+    /// </summary>
+    private async Task RegisterOpenApiSkillsAsync(CopilotChatPlanner planner, OpenApiSkillsAuthHeaders openApiSkillsAuthHeaders)
```

Comment:
[nit] Could we inline the contents of this function with "RegisterPlannerSkillsAsync" - it'd be good to keep all of this together for the first offering.